Create RC version of Device Swap (#22)
camaraproject/DeviceSwap: CHANGELOG.md

# Changelog DeviceSwap

## Table of Contents

- [r1.1](#r11)

**Please be aware that the project will have frequent updates to the main branch. There are no compatibility guarantees associated with code in any branch, including main, until it has been released. For example, changes may be reverted before a release is published. For the best results, use the latest published release.**

# r1.1
## Release Notes

This release contains the definition and documentation of
* device-swap v0.1.0-rc.1

The API definition(s) are based on
* Commonalities v0.4.0

> * Identity and Consent Management v0.2.0

Reviewer: ```suggestion
* Identity and Consent Management v0.2.1
```
(no API relevant changes, just the latest version)

* device-swap v0.1.0-rc.1

The API definition(s) are based on
* Commonalities v0.4.0
* Identity and Consent Management v0.2.0

## device-swap v0.1.0-rc.1

**device-swap v0.1.0-rc.1 is the 1st release candidate of the version 0.1.0**

- 0.1.0-rc.1 Device Swap API definition **with inline documentation**:
  - OpenAPI [YAML spec file](https://github.com/camaraproject/DeviceSwap/blob/r1.1/code/API_definitions/device-swap.yaml)
  - [View it on ReDoc](https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/camaraproject/DeviceSwap/blob/r1.1/code/API_definitions/device-swap.yaml&nocors)
  - [View it on Swagger Editor](https://editor.swagger.io/?url=https://raw.githubusercontent.com/camaraproject/DeviceSwap/blob/r1.1/code/API_definitions/device-swap.yaml)
    
## What's Changed
### Added
* Update CODEOWNERS by @ShutingQing in https://github.com/camaraproject/DeviceSwap/pull/2
* Update README.md w/ Meeting info by @wrathwolf in https://github.com/camaraproject/DeviceSwap/pull/7
* Update CODEOWNERS by @bigludo7 in https://github.com/camaraproject/DeviceSwap/pull/8
* Update MAINTAINERS.MD by @bigludo7 in https://github.com/camaraproject/DeviceSwap/pull/9

> * Update CODEOWNERS - adding Jorge Garcia Hospital by @bigludo7 in https://github.com/camaraproject/DeviceSwap/pull/12

Reviewer: I would omit these kind of "changes" from the CHANGELOG, they are internal management changes and have no value for the audience (they still documented in the "full" changelog).

Author: solved

> * Initial proposal for Device Swap API by @jgarciahospital in https://github.com/camaraproject/DeviceSwap/pull/13
> * Update README.md by @bigludo7 in https://github.com/camaraproject/DeviceSwap/pull/15
> * Clean error and description by @jgarciahospital in https://github.com/camaraproject/DeviceSwap/pull/18
> * Update device-swap description by @jgarciahospital in https://github.com/camaraproject/DeviceSwap/pull/17
> * Update APi actions descriptions by @jgarciahospital in https://github.com/camaraproject/DeviceSwap/pull/21

Reviewer: ```suggestion
* Initial proposal for Device Swap API by @jgarciahospital in https://github.com/camaraproject/DeviceSwap/pull/13
* Clean error and description by @jgarciahospital in https://github.com/camaraproject/DeviceSwap/pull/18
* Updated device-swap description by @jgarciahospital in https://github.com/camaraproject/DeviceSwap/pull/17
* Updated API actions descriptions by @jgarciahospital in https://github.com/camaraproject/DeviceSwap/pull/21
```

Author: included as suggested

### Changed
N/A
### Fixed
N/A
### Removed
N/A

## New Contributors

> * @ShutingQing made their first contribution in https://github.com/camaraproject/DeviceSwap/pull/2
> * @wrathwolf made their first contribution in https://github.com/camaraproject/DeviceSwap/pull/7
> * @bigludo7 made their first contribution in https://github.com/camaraproject/DeviceSwap/pull/8
> * @jgarciahospital made their first contribution in https://github.com/camaraproject/DeviceSwap/pull/13

Reviewer: ```suggestion
* @bigludo7 made their first contribution in https://github.com/camaraproject/DeviceSwap/pull/8
* @jgarciahospital made their first contribution in https://github.com/camaraproject/DeviceSwap/pull/13
```

Author: also cleaned
